<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bridgeGPIOWidget</class>
 <widget class="QWidget" name="bridgeGPIOWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>581</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <spacer name="mainLayoutSpacer0">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="configGroup">
      <property name="title">
       <string>Config</string>
      </property>
      <layout class="QVBoxLayout" name="configGroupLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <item>
          <spacer name="buttonLayoutSpacerLeft">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="buttonGPIO0">
           <property name="text">
            <string>GPIO0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="buttonGPIO1">
           <property name="text">
            <string>GPIO1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="buttonGPIO2">
           <property name="text">
            <string>GPIO2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="buttonGPIO3">
           <property name="text">
            <string>GPIO3</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="buttonLayoutSpacerRight">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="configLayout">
         <item>
          <spacer name="configSpacerLeft">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="modeLayout">
           <item>
            <widget class="QLabel" name="modeLabel">
             <property name="text">
              <string>Mode</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="modeBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="speedLayout">
           <item>
            <widget class="QLabel" name="speedLabel">
             <property name="text">
              <string>Speed</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="speedBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="pupdLayout">
           <item>
            <widget class="QLabel" name="pupdLabel">
             <property name="text">
              <string>Pull up/down</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="pupdBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="otLayout">
           <item>
            <widget class="QLabel" name="otLabel">
             <property name="text">
              <string>Output Type</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="otBox"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="configSpacerRight">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="configSetLayout">
         <item>
          <spacer name="configSetLayoutSpacerLeft">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonConfigSet">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="configSetLayoutSpacerRight">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="mainLayoutSpacer1">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="readGroup">
      <property name="title">
       <string>Read</string>
      </property>
      <layout class="QVBoxLayout" name="readGroupLayout">
       <item>
        <layout class="QHBoxLayout" name="stateGPIOLayout">
         <item>
          <spacer name="stateGPIOspacer0">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="stateGPIO0Layout">
           <item>
            <widget class="QLabel" name="labelReadGPIO0">
             <property name="text">
              <string>GPIO0 State</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelStateGPIO0">
             <property name="text">
              <string>SET/RESET</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="stateGPIOspacer1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="stateGPIO1Layout">
           <item>
            <widget class="QLabel" name="labelReadGPIO1">
             <property name="text">
              <string>GPIO1 State</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelStateGPIO1">
             <property name="text">
              <string>SET/RESET</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="stateGPIOspacer2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="stateGPIO2Layout">
           <item>
            <widget class="QLabel" name="labelReadGPIO2">
             <property name="text">
              <string>GPIO2 State</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelStateGPIO2">
             <property name="text">
              <string>SET/RESET</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="stateGPIOspacer3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="stateGPIO3Layout">
           <item>
            <widget class="QLabel" name="labelReadGPIO3">
             <property name="text">
              <string>GPIO3 State</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelStateGPIO3">
             <property name="text">
              <string>SET/RESET</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="stateGPIOspacer4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="readLayout">
         <item>
          <spacer name="contReadSpacerLeft">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="contReadBox">
           <property name="text">
            <string>Enable Continuous Reading</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="contReadSpacer0">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="contReadIntEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>1000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="contReadIntLabel">
           <property name="text">
            <string>Interval (ms)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="contReadSpacerRight">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="readButtonLayout">
         <item>
          <spacer name="readButtonSpacerLeft">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="readButton">
           <property name="text">
            <string>Read</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="readButtonSpacerRight">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="mainLayoutSpacer2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="writeGroup">
      <property name="title">
       <string>Write</string>
      </property>
      <layout class="QHBoxLayout" name="writeGroupLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <spacer name="writeLayoutSpacerLeft">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="writeGPIO0Layout">
         <item>
          <widget class="QLabel" name="labelGPIO0">
           <property name="text">
            <string>GPIO0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boxGPIO0"/>
         </item>
         <item>
          <widget class="QPushButton" name="buttonWriteGPIO0">
           <property name="text">
            <string>Write</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="writeLayoutSpacer0">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="writeGPIO1Layout">
         <item>
          <widget class="QLabel" name="labelGPIO1">
           <property name="text">
            <string>GPIO1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boxGPIO1"/>
         </item>
         <item>
          <widget class="QPushButton" name="buttonWriteGPIO1">
           <property name="text">
            <string>Write</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="writeLayoutSpacer1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="writeGPIO2Layout">
         <item>
          <widget class="QLabel" name="labelGPIO2">
           <property name="text">
            <string>GPIO2</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boxGPIO2"/>
         </item>
         <item>
          <widget class="QPushButton" name="buttonWriteGPIO2">
           <property name="text">
            <string>Write</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="writeLayoutSpacer2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="writeGPIO3Layout">
         <item>
          <widget class="QLabel" name="labelGPIO3">
           <property name="text">
            <string>GPIO3</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boxGPIO3"/>
         </item>
         <item>
          <widget class="QPushButton" name="buttonWriteGPIO3">
           <property name="text">
            <string>Write</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="writeLayoutSpacerRight">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="mainLayoutSpacer3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
